onnxruntime/core/providers/tensorrt/tensorrt_execution_provider.cc

```diff
@@ -108,13 +111,19 @@ bool SetDynamicRange(nvinfer1::INetworkDefinition& network, std::unordered_map<s
             case nvinfer1::DataType::kINT32:
               weight = static_cast<const int32_t*>(trt_weights.values)[k];
               break;
+#if NV_TENSORRT_MAJOR >= 10
+            case nvinfer1::DataType::kINT64:
+              weight = static_cast<const int64_t*>(trt_weights.values)[k];
```

Comment:
Converting "const int64_t" to "double" may lost data.
````
warning C4244: '=': conversion from 'const int64_t' to 'double', possible loss of data
````

Reply:
@kevinch-nv, fyi this is causing the build to fail.

Reply:
Updated